32-bit architectures: util/linux_util/linux_user.go:176:31: cannot use _Ctype_ulong(bufLen) (value of type _Ctype_ulong) as _Ctype_uint value in argument to (_Cfunc__CMalloc)

(Native) build error on 32-bit architectures (i586 and arm32):

```
[00:00:27] + go build -v -o ssh3-server cli/server/main.go
[00:00:27] text/tabwriter
[00:00:27] html
[00:00:27] internal/lazyregexp
[00:00:27] runtime/trace
[00:00:27] github.com/creack/pty
[00:00:27] runtime/pprof
[00:00:27] internal/profile
[00:00:27] ssh3/util/linux_util
[00:00:28] net/http/pprof
[00:00:28] # ssh3/util/linux_util
[00:00:28] util/linux_util/linux_user.go:176:31: cannot use _Ctype_ulong(bufLen) (value of type _Ctype_ulong) as _Ctype_uint value in argument to (_Cfunc__CMalloc)
[00:00:28] util/linux_util/linux_user.go:180:77: cannot use _Ctype_ulong(bufLen) (value of type _Ctype_ulong) as _Ctype_uint value in variable declaration
```

--- a/util/unix_util/linux_user.go
+++ b/util/unix_util/linux_user.go
@@ -140,10 +140,10 @@ func getpwnam(name string) (*User, error) {
 		bufLen = uint64(cBufLen)
 	}
 
-	buf := (*C.char)(C.malloc(C.ulong(bufLen)))
+	buf := (*C.char)(C.malloc(C.size_t(bufLen)))
 	defer C.free(unsafe.Pointer(buf))
 
-	ret, err := C.getpwnam_r(cname, cpasswd, buf, C.ulong(bufLen), &cpasswd)
+	ret, err := C.getpwnam_r(cname, cpasswd, buf, C.size_t(bufLen), &cpasswd)
 
 	if int(ret) != 0 {
 		return nil, syscall.Errno(ret)

